addons/refuel/CfgVehicles.hpp

```diff
@@ -318,13 +311,20 @@ class CfgVehicles {
         GVAR(fuelCapacity) = 80;
     };
 
-    class Van_01_fuel_base_F: Van_01_base_F {
+    class Van_01_fuel_base_F: Van_01_base_F {};
+    class C_Van_01_fuel_F: Van_01_fuel_base_F {
         transportFuel = 0; //1k
         MACRO_REFUEL_ACTIONS
         GVAR(hooks)[] = {{0.38,-3.17,-.7},{-0.41,-3.17,-.7}};
         GVAR(fuelCargo) = 2000;
     };
-
+    class I_G_Van_01_fuel_F: Van_01_fuel_base_F {
+        transportFuel = 0; //1k
+        MACRO_REFUEL_ACTIONS
+        GVAR(hooks)[] = {{0.38,-3.17,-.7},{-0.41,-3.17,-.7}};
+        GVAR(fuelCargo) = 2000;
+    };
```

Comment:
What's the point of changing the class here? New we have two vehicles (retextures) with the same config entries.